test/libsolidity/semanticTests/cleanup/exp_cleanup_nonzero_base.sol

```diff
@@ -1,7 +1,7 @@
 contract C {
     function f() public pure returns (uint8 x) {
         unchecked {
-            return uint8(0x166)**uint8(uint8(2)**uint8(8));
+            return uint8(0x16)**uint8(uint8(2)**uint8(8));
```

Comment:
Should I remove this test or maybe rename it?

Reply:
does `uint16 x= 0x166; return uint8(x)**...` work?

Reply:
Yes. I changed it.